src/libasr/ASR.asdl

```diff
@@ -97,6 +97,8 @@ symbol
         identifier original_name, access access)
     | DerivedType(symbol_table symtab, identifier name, identifier* members,
         abi abi, access access, symbol? parent)
+    | EnumType(symbol_table symtab, identifier name, identifier* members,
+        abi abi, access access, ttype type, symbol? parent)
```

Comment:
Each `EnumType` has one underlying type, say `i32`, to represent each enumerator, and this type is stored in `type`?

Reply:
Yes. Correct.